### PR TITLE
Calculate and color fault trajectory

### LINKED ISSUE #107: Mark fault trajectory
Fault in a single point often triggers cascading fault propagation throughout its neighborhood. Typically, fault trajectories span across multiple cluster elements, each of which yields an alert:

fault-trajectory-2

The goal of the task is to provide basic highlighting of fault trajectories in the graph:

1. Mark each node connected with an alert: set thick red border, fill color remains the same.
2. For each pair of marked (and connected) nodes, mark edge between them: thick red color.

### PR DESCRIPTION
**Pull request by k-jano**
Resolves #107. Calculating and coloring fault trajectory in a graph.

FaultTrajectory

### CODE REVIEW

File: src/components/Graph.scss [hunk 1]
```diff
@@ -13,6 +13,10 @@ line.link {
     stroke-width: 1;
 }
 
+line.fault {
+    stroke: $accent-color-red;
```
Comment (bzurkowski) [inline src/components/Graph.scss:17]: Let's make it thicker.

File: src/components/Graph.js [hunk 2]
```diff
@@ -322,7 +359,7 @@ export class Graph extends React.Component {
     const link = this.state.link
       .data(links, d => [d.source, d.target])
       .join('line')
-      .attr('class', 'link');
+      .attr('class', d => {return d.fault ? 'link fault' : 'link'});
```
Comment (bzurkowski) [inline src/components/Graph.js:362]: Could we also set custom fault class to nodes affected by an alert (nodes participating in fault trajectory)? For instance, set a thick red border around it - the fill color should remain the same.

**Review by bzurkowski — CHANGES_REQUESTED** [review]
@k-jano That was a fast one! 😉 I added two minor remarks for you to consider. Thanks! 👍

**Comment by k-jano** [comment]
@bzurkowski Thanks for suggestion! :+1 I made fault link and node border stroke 2 times thicker. What do you think?

FaultTrajectory2

**Review by bzurkowski — APPROVED** [review]
@k-jano It's just perfect, I love it 👍 Can't wait to test it against cascading failures in my testbed to see longer fault trajectories 😄 I'm glad you managed to contribute so much today, thanks!

**Pull request merged**